### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by hiwina**
List of things done in this project.

- [ ] Added index.html, about.html, index.css, about.css and index.js files.
- [ ] dynamic page is created for featured speakers
- [ ] Configure lint
- [ ] Created a video that explains what the project is about.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -146,14 +139,6 @@ seeMorebtn.addEventListener('click', () => {
...
 
-
-
-
-
-
-
-
-
 //     const info = document.createElement('div');
```
Comment (uwadonat) [inline index.js]: - [ ] Let's remove all commented lines to make our code more clean.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,126 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here is a link to deployed project.
+
+- [Live Demo Link](https://github.com/hiwina/Capstone-Project-1/)
+
```
Comment (uwadonat) [inline README.md]: - [ ] This link should be your deployed application. now it is pointing to your github page. Kindly fix it.

File: README.md [hunk 3]
```diff
@@ -1,40 +1,126 @@
...
+> Here is a link to loom video.
+
+- [Loom Video Link](https://www.loom.com/share/6ae631ddfe6241d7a718ad9c7a23da5e)
+
```
Comment (uwadonat) [inline README.md]: - [ ] In your presentation, you should highlight a piece of project you find interesting or you are proud of making.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @hiwina,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### Highlights
- Professional readme file ✔️
- Descriptive pull request message ✔️
- Good commit messages ✔️
almoo

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 4]
```diff
@@ -1,40 +1,126 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here is a link to deployed project.
+
+- [Live Demo Link](ttps://hiwina.github.io/Capstone-Project-1/)
```
Comment (mgmediaweb) [inline README.md]: Kindly, I suggest you follow the previous reviewer's advice, the LIVE DEMO link, currently connects to your repository, and it should point to the deployed version of your site.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @hiwina , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

almost5

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @hiwina,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 


approval1
# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**